test/uk/gov/hmrc/economiccrimelevyregistration/utils/StringUtilsSpec.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.economiccrimelevyregistration.utils
+
+import uk.gov.hmrc.economiccrimelevyregistration.base.SpecBase
+import uk.gov.hmrc.economiccrimelevyregistration.utils.StringUtils._
+class StringUtilsSpec extends SpecBase {
+
+  "removeWhitespace" should {
+    "remove any whitespace from a given string" in {
+      val result = "ABC 123 DEF".removeWhitespace
```

Comment:
Being pedantic now, but should the original string include whitespace at the beginning and end and also multiple consecutive whitespace characters as well as a mix of other characters present in the whitespace class of characters to prove that all whitespace is removed?